src/styles/utils/_mixins.scss

```
@@ -0,0 +1,10 @@
+@mixin circle ($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin square ($width, $height) {
+  width: $width;
+  height: $height;
+}
```

це вже не квадрат. Квадрат може приймати довжину
```suggestion
@mixin square ($size) {
  width: $size;
  height: $size;
}
```